src/Appwrite/Utopia/Response/Model/Migration.php

```diff
@@ -86,6 +86,12 @@ public function __construct()
                 'default' => [],
                 'example' => [],
             ])
+            ->addRule('options', [
+                'type' => self::TYPE_JSON,
+                'description' => 'Migration options used during the migration process.',
+                'default' => [],
+                'example' => '{"bucketId": "exports", "notify": false}',
+            ])
```

Comment:
why do we need this on response? we don't keep this for realtime though 🤔

Reply:
Removing from realtime is a bug that I was just checking on console, thanks :grin: